deploy/charts/external-secrets/templates/cert-controller-rbac.yaml

```diff
@@ -24,6 +24,14 @@ rules:
     - "get"
     - "list"
     - "watch"
+  - apiGroups:
+    - "admissionregistration.k8s.io"
+    resources:
+    - "validatingwebhookconfigurations"
```

Comment:
Both of these are already defined up above. Why not just add the resourceNames there?

Reply:
Thank you for your reminder, the correction has been made.